Assets/HoloToolkit/SpatialMapping/Tests/Support/Materials/lambert1.mat

```diff
@@ -1,127 +0,0 @@
-%YAML 1.1
```

Comment:
Will this and defaultMat.mat keep regenerating themselves when the fbxs are imported?